shasta/env.py

```diff
@@ -74,8 +74,12 @@ def step(self, action):
         observation, info = self.experiment.get_observation(raw_data, self.core)
         done = self.experiment.get_done_status(observation, self.core)
         reward = self.experiment.compute_reward(observation, self.core)
-
-        return observation, reward, done, info
+        if hasattr(self.experiment, 'get_truncated_status'):
```

Comment:
Let's not have a special condition. Let's make it as requirement in the base experiment.

Reply:
done. new commit makes the truncated function a requirement.